modules/@apostrophecms/doc-type/ui/apos/components/AposDocContextMenu.vue

```diff
@@ -417,15 +418,15 @@ export default {
       });
       Object.assign(doc, complete);
 
-      apos.bus.$emit('admin-menu-click', {
-        itemName: `${this.moduleName}:editor`,
-        props: {
-          copyOf: {
-            ...this.current || doc,
-            _id: doc._id
-          }
-        }
+      await apos.modal.execute(doc._aposEditorModal || this.moduleOptions.components.editorModal, {
```

Comment:
A more direct and modern way to do the same thing. Allows us to use `_aposEditorModal`.